R/plot.vpts.R

```diff
@@ -105,7 +105,10 @@ plot.vpts <- function(x, xlab = "time", ylab = "height [m]", quantity = "dens",
                       n_color=1000, palette = NA,
                       ...) {
   stopifnot(inherits(x, "vpts"))
-  stopifnot(quantity %in% names(x$data))
+  assertthat::assert_that(
+    quantity %in% names(x$data),
+    msg = "quantity needs to be one of `dens`, `eta`, `dbz` or `DBZH`"
```

Comment:
this message is too specific, the user can plot any of the available quantities, e.g. also speed, direction etc. Message should simply warn if user request a quantity that does not exist.